eng/scripts/Invoke-DependencyCheck.ps1

```diff
@@ -0,0 +1,77 @@
+Param(
+    [string] $newPackageDirectory
+)
+
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+
+$ignoreCheck = "sdk/synapse/azartifacts", "sdk/template/aztemplate", "sdk/azcore", "sdk/azidentity"
+
+$sdks = Get-AllPackageInfoFromRepo
+
+$sdkRoot = Join-Path $RepoRoot "sdk"
+
+# Get all existed packages
+$packagesImport = ""
+$newPackage = ""
+foreach ($sdk in $sdks)
+{
+    if ($sdk.Name -like "*internal*" -or $sdk.Name -in $ignoreCheck)
+    {
+        continue
+    }
+    if ((Resolve-Path $sdk.DirectoryPath).Path -eq (Resolve-Path (Join-Path $sdkRoot $newPackageDirectory)).Path)
+    {
+        $newPackage = $sdk.Name
```

Comment:
I would just make the special relative link import here instead of doing it later below.

Reply:
Changed.